docs/local-lxd/how-to/integrate-with-cos.md

```diff
@@ -0,0 +1,50 @@
+# How to integrate with COS
+
+This guide demonstrates the process of integrating with the [Canonical Observability Stack (COS)](https://charmhub.io/topics/canonical-observability-stack) using the optional `cos-agent` integration provided by this charm.
+
+The `cos-agent` integration can be consumed by the [grafana-agent](https://charmhub.io/grafana-agent) charm, which is responsible for transmitting logs, Prometheus metrics, and Grafana dashboards to the COS stack.
+
+> NOTE: The Github Runner charm and `grafana-agent` charm function as machine charms, while the COS stack comprises Kubernetes charms. Therefore, establishing [cross-model integrations](https://juju.is/docs/juju/manage-cross-model-integrations) is necessary, along with potential firewall rule configurations to allow inter-model traffic.
```

Comment:
```suggestion
> NOTE: The Github Runner charm and `grafana-agent` charm function as machine charms, while the COS stack contains Kubernetes charms. Therefore, establishing [cross-model integrations](https://juju.is/docs/juju/manage-cross-model-integrations) is necessary, along with potential firewall rule configurations to allow inter-model traffic.
```
If "contains" does not convey the correct meaning, then I suggest using "is comprised of" instead.